### PR TITLE
Add missing API doc for create_distributed_restore_point

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

The docs for this API call got lost in the restructure somehow

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Originally added in https://github.com/timescale/docs.timescale.com-content/pull/673

### CODE REVIEW

**Comment by Loquacity** [comment]
image

File: api/create_distributed_restore_point.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+|-|-|
+|`name`|The restore point name|
+
+## Returns
+
+|Column|Type|Description|
+|-|-|-|
+|`node_name`|NAME|Node name, or `NULL` for access node|
+|`node_type`|TEXT|Node type name: `access_node` or `data_node`|
+|`restore_point`|[PG_LSN][pg-lsn])|Restore point log sequence number|
```
Comment (pmwkaa) [inline api/create_distributed_restore_point.md:29]: I guess we don't need that extra bracket here:  [PG_LSN][pg-lsn]`)`

**Review by pmwkaa — APPROVED** [review]
Looks good, thanks!

File: api/create_distributed_restore_point.md (hunk 1 above)
Comment (Loquacity) [reply api/create_distributed_restore_point.md:29]:
```suggestion
|`restore_point`|[PG_LSN][pg-lsn]|Restore point log sequence number|
```


Good spot!

**Pull request merged**